process/interceptors/factory/interceptedDataFactory_test.go

```diff
@@ -177,6 +179,23 @@ func TestInterceptedDataFactory_CreateInterceptedTxShouldWork(t *testing.T) {
 	assert.True(t, ok)
 }
 
+func TestInterceptedDataFactory_CreateInterceptedUnsignedTxShouldWork(t *testing.T) {
+	t.Parallel()
+
+	marshalizer := &mock.MarshalizerMock{}
+	emptyTx := &smartContractResult.SmartContractResult{}
+	emptyTxBuff, _ := marshalizer.Marshal(emptyTx)
+
+	idf, _ := factory.NewInterceptedDataFactory(createMockArgument(), factory.InterceptedUnsignedTx)
+
+	instance, err := idf.Create(emptyTxBuff)
+
+	assert.NotNil(t, instance)
```

Comment:
remove empty line and add empty line before type assertion

Reply:
made modifications to conform to arrange-test-assert structure